Fastfile

```diff
@@ -4,95 +4,80 @@ default_platform :ios
 
 platform :ios do
 
-  # Run this before doing anything else
+  desc "Run this before doing anything else"
+  desc "Perform project configurations validations."
   before_all do
-
-    desc "Perform project configurations validations."
     validate
-    
```

Comment:
Sorry, but I prefer the old way. The commented line is about when the method is called (before_all, after_all, etc). If not, we can remove it directly.
And the description inside, if I understand correctly is what will be printed when running this, and it will say all the steps, all the things you are doing in that function.

Reply:
I think the reason why pol did this is because using a desc makes this show up in the fastlane documentation. I also like it better as it was before. @pablog confirm this.

Reply:
Exactly, as capi says I did it for that, but anyway I don't consider they're useful comments.

You suggest removing them or leaving them as comments or as @dtilve says (I prefer this) using `UI` to inform the user what the lane is doing.

Reply:
Hmm, maybe just a small desc for documentation and then the UI to inform the user I think would be great.